common.lic

```diff
@@ -391,42 +391,50 @@ module DRC
     g
   end
 
-  def play_song?(settings, song_list, worn = true, skip_clean = false)
+  def play_song?(settings, song_list, worn = true, skip_clean = false, climbing = false)
     instrument = worn ? settings.worn_instrument : settings.instrument
     UserVars.song = song_list.first.first unless UserVars.song
+    UserVars.climbing_song = @song_list.first.first unless UserVars.climbing_song || !climbing
```

Comment:
❓ Why does this have `|| !climbing`? It seems to me like it should be sufficient to just check if the UserVar is set

Reply:
When I started this I was thinking none of the climbing logic should trigger unless <climbing> was true.  Now that I understand it better I don't think it matters.  Since that unless is really confusing I'll remove it prior to another couple tests.